store/conf/attrs/zimbra-attrs.xml

```diff
@@ -2690,7 +2690,7 @@ TODO - add support for multi-line values in globalConfigValue and defaultCOSValu
 </attr>
 
 <attr id="512" name="zimbraBackupMode" type="enum" value="Standard,Auto-Grouped" cardinality="single" optionalIn="globalConfig,server" flags="serverInherited">
-  <globalConfigValue>Standard</globalConfigValue>
+  <globalConfigValue>Auto-Grouped</globalConfigValue>
```

Comment:
for upgrade scenario, will this change applied automatically or do we need to write some migration script?

Reply:
So by default, it will be `Auto-Grouped`, and what for the existing setups?

Reply:
whenever we upgrade to the old setup, local config, and LDAP values should not change. 

If we want to apply new value to old setup, then we need a script.